packages/terra-consumer-layout/package.json

```diff
@@ -38,7 +38,9 @@
     "terra-i18n": "^1.5.0",
     "terra-icon": "^1.5.0",
     "terra-overlay": "^1.7.0",
-    "terra-responsive-element": "^1.7.0"
+    "terra-responsive-element": "^1.7.0",
+    "terra-alert": "^1.9.1",
+    "js-cookie": "2.1.3"
```

Comment:
Only the UI component should be in this repo. Cookies are a detail how state is a manager and should be handled by the app consuming the component.

Reply:
Yes ,I have deleted cookies. forgot to remove it here